esphomeyaml/components/my9231.py

```diff
@@ -0,0 +1,51 @@
+import voluptuous as vol
+
+import esphomeyaml.config_validation as cv
+from esphomeyaml import pins
+from esphomeyaml.components import output
+from esphomeyaml.const import (CONF_DATA_PIN, CONF_CLOCK_PIN, CONF_NUM_CHANNELS,
+                               CONF_NUM_CHIPS, CONF_BIT_DEPTH, CONF_ID,
+                               CONF_UPDATE_ON_BOOT)
+from esphomeyaml.helpers import (gpio_output_pin_expression, App, Pvariable,
+                                 add)
+
+MY9231OutputComponent = output.output_ns.namespace('MY9231OutputComponent')
+
+
+MY9231_SCHEMA = vol.Schema({
+    cv.GenerateID(): cv.declare_variable_id(MY9231OutputComponent),
+    vol.Required(CONF_DATA_PIN): pins.gpio_output_pin_schema,
+    vol.Required(CONF_CLOCK_PIN): pins.gpio_output_pin_schema,
+    vol.Optional(CONF_NUM_CHANNELS): vol.All(vol.Coerce(int),
+                                             vol.Range(3, 1020)),
+    vol.Optional(CONF_NUM_CHIPS): vol.All(vol.Coerce(int),
+                                          vol.Range(1, 255)),
+    vol.Optional(CONF_BIT_DEPTH): vol.All(vol.Coerce(int),
+                                          vol.In([8, 12, 14, 16])),
```

Comment:
`vol.In` emits quite bad error messages, use:

```suggestion
                                          cv.one_of(8, 12, 14, 16)),
```

Reply:
Thank, you! fixed